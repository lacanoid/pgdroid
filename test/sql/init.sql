CREATE EXTENSION dbms cascade;
\pset tuples_only

\dx+ dbms
