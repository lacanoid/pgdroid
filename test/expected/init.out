CREATE EXTENSION dbms cascade;
NOTICE:  installing required extension "plperlu"
\pset tuples_only
\dx+ dbms
 event trigger ddl_log
 function attribute_names(regclass,smallint[])
 function attribute_types(regclass,smallint[])
 function describe(regclass)
 function get_proc_info(text,text)
 function json_save(regclass,text,boolean,boolean)
 function primary_key(regclass)
 function trigger_ddl()
 function trigger_dml()
 sequence dml_log_seq_seq
 table ddl_log
 table dml_log
 type sql_identifier
 view catalog_usage
 view unique_keys

